doc/api/api_changes/2015-04-18-drawingarea.rst

```diff
@@ -0,0 +1,8 @@
+'OffsetBox.DrawingArea' no longer accepts the 'clip' keyword argument
```

Comment:
The title is no longer accurate.  Should be 'OffsetBox.DrawingArea now respects `clip`' or something to that effect.